course-info.yaml

```diff
@@ -1,118 +1,76 @@
 type: marketplace
-title: 'C++Basics'
+title: C++ Basics
 language: English
+summary: "<p>C++ programming is a superpower! Master memory management, build fast\
+  \ programs, and create a 2D game from scratch. No experience needed—this course\
+  \ makes C++ fundamentals beginner-friendly and fun."
```

Comment:
Is it possible to keep the summary in human-readable format, as it was done before?

Reply:
The plugin automatically formats YAML files. I don't think it's worth fighting this. In this case we will constantly have a diff with formatting updates.